README.md

```diff
@@ -70,6 +71,7 @@ Options:
   -a, --all           Includes pre and post scripts on the list        [boolean]
   -A, --autocomplete  Starts in autocomplete mode                      [boolean]
   -d, --debug         Prints to stderr any internal error              [boolean]
+  -D, --descriptions  Displays the descriptions of each script         [boolean]
```

Comment:
I think it would be nicer to have `-d` as alias for that option.

But in this case we would need to rename the `--debug` alias.  That means a breaking change and would cause a major release. What’s your opinion on this?

Reply:
👍 to swap both and make it a major release!